stamina-testkit/README.md

```diff
@@ -0,0 +1,59 @@
+# stamina-testkit
+
+*stamina-testkit* contains utilities for generating tests for persisters based on example events.
+
+## Usage
+
+Currently only scalatest is supported. Tests can be generated based on example events like this:
+
+```scala
+import stamina._
+import stamina.testkit._
+
+import BasketActor._
+
+import common.DateTimeSupport._
+
+class BasketActorAkkaSerializationSpec extends UnitSpec with ScalaTestKit {
+  val persisters = Persisters(BasketActorAkkaSerialization.persisters)
+
+  val timestamp = "2015-01-01T12:34:56.000Z".toDateTime
+  val code = "EXAMPLE_CODE"
+
+  "The BasketActor akka serialization" should {
```

Comment:
`"The BasketActor persister" should {` since this has nothing to do with akka serialization